Dashboard Controls: Display dashboard links on the right side of the toolbar

harisrozajac opened the pull request:
Fixes an issue where dashboard links appeared on the left side of the navigation toolbar, and restores the previous behavior, placing them back on the right side.

With this fix, I wanted to make sure I didn't break the functionality implemented in https://github.com/grafana/grafana/pull/113765 and https://github.com/grafana/grafana/pull/113927

Demo:

https://github.com/user-attachments/assets/bdd4c06c-547e-4b55-85c4-57db5d7bc8a1


Fixes: https://github.com/grafana/support-escalations/issues/19594

Please check that:
- [ ] It works as expected from a user's perspective.

harisrozajac committed:
have dashboard links on the right side

oscarkilhed reviewed (commented):
Unfortunately if you add a lot of links they extend outside of the viewport to the left.

oscarkilhed committed:
Lets try a compromise (#114389)

Try to compromise on white space. Having a lot of links will create unessecary empty space under the time controls, but it is necessary if we want to be able to have the links on the right

harisrozajac commented:
@oscarkilhed thanks for implementing this! I've merged your PR to this one, so feel free to merge if everything looks ok.

Sergej-Vlasov committed:
Merge branch 'main' into haris/dash-links-main
